.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "src/libs/redox"]
 	path = src/libs/redox
 	url = https://github.com/dcrankshaw/redox.git
+[submodule "src/libs/rapidjson"]
+	path = src/libs/rapidjson
+	url = https://github.com/giulio-zhou/rapidjson.git
```

Comment:
Why add this as a submodule? It's a header-only library. Just copy the headers into `src/libs` the same way we did for the HTTP server or ZMQ. Make sure to include the license when you copy the headers over.